website/docs/r/budget.html.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: "aws"
+page_title: "AWS: aws_budgets_budget"
+sidebar_current: "docs-aws-resource-budget"
+description: |-
+  Provides a budget resource.
+---
+
+# aws_budgets_budget
+
+Provides a budget resource. Budgets use the cost visualisation provided by Cost Explorer to show you the status of your budgets, to provide forecasts of your estimated costs, and to track your AWS usage, including your free tier usage.
+
+## Example Usage
+
+```hcl
+resource "aws_budgets_budget" "ec2" {
+  name                  = "budget-ec2-monthly"
+  budget_type           = "spend"
+  limit_amount          = "1200"
+  limit_unit            = "dollars"
+  time_period_end       = "2087-06-15_00:00"
+  time_period_start     = "2017-07-01"
+  time_unit             = "MONTHLY"
+
+  cost_filters {
+    Service = "ec2"
+  }
+}
+```
+
+Create a budget for *$100*.
+
+```hcl
+resource "aws_budgets_budget" "cost" {
+  ...
+  budget_type  = "spend"
+  limit_amount = "100"
+  limit_unit   = "dollars"
+}
+```
+
+Create a budget for s3 with a limit of *3 GB* of storage.
+
+```hcl
+resource "aws_budgets_budget" "s3" {
+  ...
+  budget_type  = "usage"
+  limit_amount = "3"
+  limit_unit   = "GB"
+}
+```
+
+## Attributes Reference
```

Comment:
We generally list attributes after arguments and should only list attributes that aren't already arguments like `id`. 👍